### DISCUSSION TITLE
分区调整提醒

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1511] [功能建议 / Ideas]
[动物圈分区调整公告](https://www.bilibili.com/read/cv9507220)
> 1月29日将动物圈由二级分区整合升级成新的一级分区。

>   即将上线的新动物圈一级分区，将包含喵星人、汪星人、大熊猫、野生动物、爬宠、动物综合等六个二级分区。

**Comment by the1812** [comment]
<img width="100" src="https://user-images.githubusercontent.com/26504152/106378831-c2fc3780-63e2-11eb-96e7-c2a67efe55e4.png">

**Reply by Geeyun-JY3** [reply]
要不以后分区调整都发这吧🌚

**Comment by Geeyun-JY3** [comment]
[知识、数码和科技分区调整公告](https://www.bilibili.com/read/cv11530396)

**Comment by Geeyun-JY3** [comment]
自定义顶栏的主站组件好像忘了跟进？少了运动、汽车：
（2024-12-30 编辑：补一个建立运动一级分区的公告：[运动分区调整公告](https://www.bilibili.com/read/cv12139175/)）
![image](https://user-images.githubusercontent.com/49396421/139098735-14aac359-b3d7-47e0-bb23-999e76d9ab11.png)

另外提醒一下新版的顶栏相关组件，差别不小，拆得更细了：
![image](https://user-images.githubusercontent.com/49396421/139099080-349426b3-e653-4788-ab4c-d8d68865d5c2.png)

**Reply by the1812** [reply]
放映厅是没了?

**Reply by Geeyun-JY3** [reply]
[放映厅](https://www.bilibili.com/cinema/) 页面还在，不过在这里是拆成电影、电视剧、综艺、纪录片了。
放映厅页面就有这四个分类。

**Reply by the1812** [reply]
怪了, 今天看又变了
~~不会是我已经鸽了两轮分区调整了吧~~

![image](https://user-images.githubusercontent.com/26504152/143763359-414e78c6-4cc8-45af-84b1-2e02406b565f.png)

**Reply by Geeyun-JY3** [reply]
> 怪了, 今天看又变了 ~不会是我已经鸽了两轮分区调整了吧~
> 
> ![image](https://user-images.githubusercontent.com/26504152/143763359-414e78c6-4cc8-45af-84b1-2e02406b565f.png)

我这边被退出新版了，没办法验证。
但这个不就是现在顶栏的样子吗？~~又改回去了？~~

**Reply by Geeyun-JY3** [reply]
今天用着 v1 的主力浏览器被推到了

好像还是我发的第二张图的样子啊。首页能看到。

**Reply by the1812** [reply]
哦 你这么一说我才发现首页还是旧版的, 进一下新版就看到了

**Reply by Geeyun-JY3** [reply]
原版顶栏鼠标悬停首页出来的菜单里的分区入口现在改成这样了：
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/1ad05a58-60cd-4b24-8c69-a719ff97da21)
现在应该是和（首页）分区栏保持一致了：
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/da7cd176-76c2-4539-875c-2e98ba861241)

与上图所列分区相比，除了[之前提到过](https://github.com/the1812/Bilibili-Evolved/discussions/1511#discussioncomment-1674655)的[综艺页面入口](https://www.bilibili.com/variety/)（是不是单纯就是之前更新分区的时候漏了）和单独列出来的三个子分区（或子分区下的标签）入口（[VLOG](https://www.bilibili.com/v/life/daily/?tag=530003)、[搞笑](https://www.bilibili.com/v/life/funny/)、[单机游戏](https://www.bilibili.com/v/game/stand_alone/)），目前`自定义顶栏`还少了[公益](https://love.bilibili.com/)、[公开课](https://www.bilibili.com/mooc/)、[社区中心](https://www.bilibili.com/blackboard/activity-5zJxM3spoS.html)（取代了小黑屋）。是和原版顶栏保持完全一致还是仅加上缺少的入口呢？

**Reply by the1812** [reply]
需要和原版顶栏保持一致

**Reply by the1812** [reply]
坏, 之前写的更新脚本也失效了, 那还是手动补一补缺少的吧

**Reply by Geeyun-JY3** [reply]
[综艺页面入口](https://www.bilibili.com/variety/)和单独列出来的三个子分区（或子分区下的标签）入口（[VLOG](https://www.bilibili.com/v/life/daily/?tag=530003)、[搞笑](https://www.bilibili.com/v/life/funny/)、[单机游戏](https://www.bilibili.com/v/game/stand_alone/)）还是没加啊……

**Reply by the1812** [reply]
综艺这个页面貌似和其他分区不太一样, 源码里找不到分区 ID, 其下也没有子分区 (进页面里选子分区也会直接跳到电视剧区), 看起来并不是独立的分区. 按理来说应该和 VLOG 一起放后面, 不知道 b 站为啥放在电视剧旁边

**Reply by Geeyun-JY3** [reply]
> 综艺这个页面貌似和其他分区不太一样, 源码里找不到分区 ID, 其下也没有子分区 (进页面里选子分区也会直接跳到电视剧区), 看起来并不是独立的分区. 按理来说应该和 VLOG 一起放后面, 不知道 b 站为啥放在电视剧旁边

这很好理解，这就是版权内容首页连着放一起，无所谓是不是真的独立分区。

**Reply by Geeyun-JY3** [reply]
这个分区排序目前是怎么排的，综艺这样的版权内容分区入口和单独列出来的三个子分区（或子分区下的标签）入口放一起总觉得有些反直觉……
![image](https://github.com/user-attachments/assets/2609391a-68a5-432e-9f9f-581887e60bfb)

**Reply by the1812** [reply]
综艺也是子分区, 我认为和 VLOG 这些子分区排一起没啥问题, 放在前面的都是大分区
